package_schemas/general_data/2_restricted_schema.json

```diff
@@ -51,6 +51,14 @@
             "private": false,
             "visibility_classes": ["hidden"]
         },
+        {
+            "field_name": "org_to_allow_transfer_to",
+            "label": "Transfer to",
+            "preset": "org_to_allow_transfer_to",
+            "private": false,
+            "visibility_classes": ["hidden"],
```

Comment:
I'm thinking this line could go into the preset "org_to_allow_transfer_to".  Where have you defined that preset?  in ckanext-unaids/ckanext/unaids/presets.json?